addons/website_form/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -22,12 +21,15 @@
                     font-family: $o-we-sidebar-content-field-input-font-family;
                 }
                 tr {
+                    display: flex;
                     border: 1px solid rgba(white, 0.1);
                     border-left: none;
                     border-right: none;
                 }
                 td {
+                    flex-grow: 1;
                     &:first-child, &:last-child {
+                        flex-grow: 0;
```

Comment:
Good job @bvr-odoo :clap: I strongly prefer this solution!

What would you think if in addition to your change, we remove the size of 28px and make:

```scss
td {
       flex-grow: 5;
       &:first-child, &:last-child {
              flex-grow: 1;
       }
...
```
That way the we-list will be fully responsive... unless it causes a problem that I don't see :eyes:

Reply:
> Good job @bvr-odoo clap I strongly prefer this solution!
> 
> What would you think if in addition to your change, we remove the size of 28px and make:
> 
> ```css-scss
> td {
>        flex-grow: 5;
>        &:first-child, &:last-child {
>               flex-grow: 1;
>        }
> ...
> ```
> 
> That way the we-list will be fully responsive... unless it causes a problem that I don't see eyes

I think we must keep these 28px, for me the buttons should not be responsive as long as the input is. Look here with your suggestion =>

[we-list-02.webm](https://user-images.githubusercontent.com/52911687/222658185-b71c7bfd-80a2-4019-8da5-46fd3d6ba87b.webm)

I prefer like this, the input adapts but the buttons do not change =>

[we-list-01.webm](https://user-images.githubusercontent.com/52911687/222657989-791ffe4d-f11e-47eb-89c6-86c4ec78c3b6.webm)